extraction/src/test/scala/org/broadinstitute/monster/encode/extraction/EncodeExtractionPipelineBuilderSpec.scala

```diff
@@ -0,0 +1,264 @@
+package org.broadinstitute.monster.encode.extraction
+
+import better.files.File
+import org.broadinstitute.monster.common.PipelineBuilderSpec
+import org.broadinstitute.monster.common.msg.MsgOps
+import upack._
+
+import scala.collection.mutable
+
+object EncodeExtractionPipelineBuilderSpec {
+  val fakeIds = 1 to 1
```

Comment:
Is the idea that this range could be made wider in the future? I am trying to understand the benefit of using a range when it's only `1 to 1`.

Reply:
Yeah lol that's the only reason I kept it as is; if we think that's not worth it then I can certainly change it.

Reply:
I'd get rid of these for all the objects with only one result, for clarity